example/cumulus-tf/main.tf

```diff
@@ -55,6 +57,9 @@ module "cumulus" {
   vpc_id            = var.vpc_id
   lambda_subnet_ids = var.subnet_ids
 
+  rds_security_group            = local.rds_security_group
```

Comment:
So the security group comes from the cluster module, then is input to data persistence module, then is read as output from the data persistence module. I guess it can't come from the cluster module directly without adding another remote state dependency here, which we don't want

Reply:
Right - given the Cluster module is potentially user supplied.     Also remember this is `cumulus-tf`, so this is for *us* - it's entirely possible for DAACs/Cirrus to bring in a remote state into their Core setup independently from data persistence.